### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by ss-o-furda**
related: #1748

### CODE REVIEW

**Review by MariiaRomanuik — COMMENTED** [review]
You've done a great job in translation. l like it) Here are couple comments l would suggest

**Comment by tiangolo** [comment]
📝 Docs preview for commit 43faacc1f83af8cbc17e434cdd88e3d3475049d2 at: https://649a25c61636a41a28534cdf--fastapi.netlify.app

File: docs/uk/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+```hl_lines="4"
+INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+Цей рядок показує URL-адресу, де запущено ваш додаток, на вашому локальному комп’ютері.
+
+### Перевірте
+
+Відкрийте свій браузер за адресою <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/first-steps.md:44]:
```suggestion
Відкрийте у своєму браузері цю адресу <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
```

File: docs/uk/docs/tutorial/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+#### "Схема"
+
+"Схема" - це визначення або опис чогось. Не код, який це реалізує, а просто абстрактний опис.
+
+#### API "схема"
+
+У цьому випадку, <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> це специфікація, яка диктує, як визначити схему вашого API.
+
+Це визначення схеми включає ваші шляхи API, можливі параметри, які вони беруть, тощо.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/first-steps.md:80]:
```suggestion
Це визначення схеми включає ваші шляхи API, можливі параметри, які вони приймають, тощо.
```

File: docs/uk/docs/tutorial/first-steps.md [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+
+`@app.get("/")` повідомляє **FastAPI**, що функція прямо нижче відповідає за обробку запитів, які надходять до:
+
+* шляху `/`
+* використовуючи <abbr title="HTTP GET метод"><code>get</code> операцію</abbr>
+
+!!! info "`@decorator` Info"
+    Синтаксис `@something` у Python називається "декоратором".
+
+    Ви розміщуєте його поверх функції. Як гарний декоративний капелюх (я здогадуюсь, звідки пішов цей термін).
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/first-steps.md:256]:
```suggestion
    Ви розміщуєте його поверх функції. Як гарний декоративний капелюх (мабуть, звідси пішов цей термін).
```

File: docs/uk/docs/tutorial/first-steps.md [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+
+```Python hl_lines="8"
+{!../../../docs_src/first_steps/tutorial001.py!}
+```
+
+Ви можете повернути `dict`, `list`, одиничні значення як `str`, `int` тощо.
+
+Ви також можете повернути моделі Pydantic (докладніше про це ви дізнаєтеся пізніше).
+
+Існує багато інших об’єктів і моделей, які автоматично конвертуються в JSON (включаючи ORM тощо). Спробуйте використовувати свої улюблені, імовірно, що вони вже підтримуються.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/first-steps.md:325]:
```suggestion
Існує багато інших об’єктів і моделей, які автоматично конвертуються в JSON (включаючи ORM тощо). Спробуйте використовувати свої улюблені, дуже ймовірно, що вони вже підтримуються.
```

**Review by romashevchenko — CHANGES_REQUESTED** [review]
Hi, 
I'm a part of the DreamProIT Team and a native speaker. I have reviewed this translation and made some suggestions. I hope they will make the text better.

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**